include/error.h

```diff
@@ -51,4 +51,7 @@
 #define FAILURE		-1
 #endif
 
+#define RET_OK(x)	((x) == SUCCESS)
+#define RET_FAIL(x)	((x) != SUCCESS)
```

Comment:
Does it make it more readable? I feel like more often than not abstracting things can create confusion. That said I do not oppose the change, I would just wish to know if the majority of us see improvement here.

Reply:
I will put this in other pull request

Reply:
I wouldn't define this macros. They might create confusion.